specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-07-02-preview/managedClusters.json

```diff
@@ -6795,6 +6799,16 @@
       },
       "description": "The security settings of an agent pool."
     },
+    "AgentPoolGPUPProfile": {
+      "type": "object",
+      "properties": {
+        "installGPUDrivers": {
+          "type": "boolean",
+          "title": "Whether to auto install GPU drivers or not.",
+          "description": "The default value is true. GPU Driver Installation can only be set true when VM has associated GPU resouce. It is intended as an opt-out feature for users who want to install GPU drivers themselves to avoid any installations."
```

Comment:
typo, should be 'resource'. The SpellCheck pipeline catches it.

Reply:
Fixed